### PR TITLE
Add and update doc links

### PR DESCRIPTION
**Pull request by jrobsonchase**
More or less the same change as ngrok/ngrok-go#133.

Add the final destination links. ~I'll run a permalinking script after we've~
~agreed on the permalinks themselves.~ Per the updates in ngrok/ngrok-go#133, Product will handle the redirects, and these links are final.

### CODE REVIEW

**Comment by jrobsonchase** [comment]
Current dependencies on/for this PR:
* main
  * **PR #121** <a href="https://app.graphite.dev/github/pr/ngrok/ngrok-rust/121" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ngrok/ngrok-rust/121?utm_source=stack-comment).

File: ngrok/src/config/http.rs [hunk 1]
```diff
@@ -66,6 +67,33 @@ impl FromStr for Scheme {
     }
 }
 
+/// Restrictions placed on the origin of incoming connections to the edge.
+#[derive(Clone, Default)]
+pub(crate) struct UaFilter {
+    /// Rejects connections that do not match the given CIDRs
+    pub(crate) allow: Vec<String>,
+    /// Rejects connections that match the given CIDRs and allows all other CIDRs.
```
Comment (bobzilladev) [inline ngrok/src/config/http.rs:75]: Noticed these comments were never switched from CIDR to UA regexes

**Review by bobzilladev — APPROVED** [review]
All links correspond and successful. One comment about an old copy pasta, otherwise lgtm

File: ngrok/src/config/http.rs (hunk 1 above)
Comment (jrobsonchase) [reply ngrok/src/config/http.rs]: I just never merged the PR to change it :sweat_smile:

**Pull request merged**